src/patternfly/components/DataList/data-list.scss

```diff
@@ -114,53 +128,58 @@
   @include pf-t-light;
 
   list-style-type: disc;
-  background-color: var(--pf-c-data-list--BackgroundColor);
   border-top: var(--pf-c-data-list--BorderTopWidth) solid var(--pf-c-data-list--BorderTopColor);
   border-bottom: var(--pf-c-data-list--BorderBottomWidth) solid var(--pf-c-data-list--BorderBottomColor);
 }
 
 // li
 .pf-c-data-list__item {
-  position: relative;
   display: flex;
   flex-direction: column;
+  background-color: var(--pf-c-data-list--BackgroundColor);
 
-  // left border treatment base
-  // color is transparent by default
-  &::before {
-    position: absolute;
-    top: var(--pf-c-data-list__item--before--Top);
-    bottom: 0;
-    left: 0;
-    width: var(--pf-c-data-list__item--before--Width);
-    height: var(--pf-c-data-list__item--before--Height); // remove after breaking change period
-    content: "";
-    background-color: var(--pf-c-data-list__item--before--BackgroundColor);
-    transition: var(--pf-c-data-list__item--before--Transition);
+  @media (min-width: $pf-global--breakpoint--sm) {
+    --pf-c-data-list__item--before--Top: var(--pf-c-data-list__item-item--before--Top);
```

Comment:
Should this be `var(--pf-c-data-list__item--item--before--Top)`? 

The second `--item--` preceded with two dashes?

Reply:
yep! but it was that way before, so changing it would be a breaking change. we need to update a lot of these vars in a breaking change release. I've made a note of that here - https://github.com/patternfly/patternfly-next/issues/2305